libs/cpa/ngx/data-access/src/lib/services/response/response.service.spec.ts

```diff
@@ -1,25 +1,53 @@
 import { TestBed } from '@angular/core/testing';
-import { HttpClientTestingModule } from '@angular/common/http/testing';
-
-import { EnvironmentModule, env } from '@tamu-gisc/common/ngx/environment';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
 
+import { EnvironmentService } from '@tamu-gisc/common/ngx/environment';
 import { ResponseService } from './response.service';
 
+const spyEnvironment = {
+  value: jest.fn()
+};
+
 describe('ResponseService', () => {
-  beforeEach(() =>
+  let service: ResponseService;
+  let httpTestingController: HttpTestingController;
+
+  beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [EnvironmentModule, HttpClientTestingModule],
+      imports: [HttpClientTestingModule],
       providers: [
+        ResponseService,
         {
-          provide: env,
-          useValue: { api_url: 'https://' }
+          provide: EnvironmentService,
+          useValue: spyEnvironment
         }
       ]
-    })
-  );
+    });
+    httpTestingController = TestBed.inject(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('createResponse()', () => {
+    const dummyStringValue = 'Foo';
+    const dummyResponse = [
+      {
+        name: 'Foo',
+        workshops: []
+      }
+    ];
```

Comment:
Incorrect input and return type. I realize the service method being tested doesn't have a very useful implicit return type, but you are encouraged to look at the implementation and determine what kind of response comes back, if any, and have more accurate tests.